Update error-catalog with tenant management error codes (#2260)
wso2/docs-is: en/docs/references/error-catalog.md

    <th>Error Message</th>
    <th>Possible Cause</th>
  </thead>
  <tbody>
  <tr>
    <td>TM-60000</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>Provided email is empty.</td>
  </tr>
  <tr>
    <td>TM-60001</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>Wrong characters in the email.</td>
  </tr>
  <tr>
    <td>TM-60002</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>Invalid email address is provided.</td>
  </tr>
  <tr>
    <td>TM-60003</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>You can not use a registry reserved word as a tenant domain. Please choose a different one.</td>
  </tr>
  <tr>
    <td>TM-60004</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>Provided domain name is empty.</td>
  </tr>
  <tr>
    <td>TM-60005</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>You should have an extension to your domain.</td>
  </tr>
  <tr>
    <td>TM-60006</td>
    <td>400</td>
    <td>Unable to add tenant.</td>
    <td>Invalid domain. Domain should not start with '.'</td>
  </tr>
  <tr>
    <td>TM-60007</td>
    <td>400</td>
    <td>Unable to add tenant.</td>

>     <td>The tenant domain %s contains one or more illegal characters. The valid characters are lowercase letters, numbers, '.', '-' and '_'.</td>

Reviewer: ```suggestion
    <td>The tenant domain contains one or more illegal characters. The valid characters are lowercase letters, numbers, '.', '-' and '_'.</td>
```

  </tr>
  <tr>
    <td>TM-60008</td>
    <td>400</td>
    <td>Unable to add tenant.</td>

>     <td>User name : %s exists in the system. Please pick another user name for tenant Administrator.</td>

Reviewer: ```suggestion
    <td>User name exists in the system. Please pick another user name for tenant administrator.</td>
```

  </tr>
  <tr>
    <td>TM-60009</td>
    <td>400</td>
    <td>Unable to add tenant.</td>

>     <td>A tenant with same domain %s already exist. Please use a different domain name.</td>

Reviewer: ```suggestion
    <td>A tenant with same domain already exists. Please use a different domain name.</td>
```

  </tr>
  <tr>
    <td>TM-60010</td>
    <td>400</td>
    <td>Invalid Request</td>
    <td>Limit should not be negative.</td>
  </tr>
  <tr>
    <td>TM-60011</td>
    <td>400</td>
    <td>Invalid Request</td>
    <td>Offset should not be negative.</td>
  </tr>
  <tr>
    <td>TM-60012</td>
    <td>400</td>
    <td>Invalid Request</td>
    <td>Required parameter owner is not specified.</td>
  </tr>
  <tr>
    <td>TM-60013</td>
    <td>400</td>
    <td>Invalid Request</td>

>     <td>Required parameter %s is not specified.</td>

Reviewer: ```suggestion
    <td>Required parameter(s) is not specified.</td>
```

  </tr>
  <tr>
    <td>TM-60014</td>
    <td>400</td>
    <td>Unable to retrieve tenant.</td>
    <td>Tenant cannot be found for the provided id: %s.</td>
  </tr>
  <tr>
    <td>TM-60015</td>
    <td>400</td>
    <td>Unable to retrieve tenant.</td>

>     <td>Tenant cannot be found for the provided domain: %s.</td>

Reviewer: ```suggestion
    <td>Tenant can not be found for the provided domain(s).</td>
```

  </tr>
  <tr>
    <td>TM-60016</td>
    <td>400</td>
    <td>Error while deleting the tenant metadata.</td>

>     <td>Tenant deletion property Tenant.TenantDelete is not enabled in carbon.xml file./td>

Reviewer: This description seems a bit unclear. Is `Tenant.TenantDelete` a property name? If so, shall we enclose it in <code>  tags?

  </tr>
  <tr>
    <td>TM-65001</td>
    <td>500</td>
    <td>Unable to list existing tenants.</td>
    <td>Server encountered an error while listing the tenants.</td>
  </tr>
  <tr>
    <td>TM-65002</td>
    <td>500</td>
    <td>Unable to add tenant.</td>
    <td>Server encountered an error while adding the tenant.</td>
  </tr>
  <tr>
    <td>TM-65003</td>
    <td>500</td>
    <td>Unable to retrieve tenant.</td>

>     <td>Server encountered an error while retrieving the tenant for identifier %s.</td>

Reviewer: ```suggestion
    <td>Server encountered an error while retrieving the tenant for identifier(s).</td>
```

  </tr>
  <tr>
    <td>TM-65004</td>
    <td>500</td>
    <td>Error while updating the tenant.</td>

>     <td>Server encountered an error while the tenant life cycle status activated: %s .</td>

Reviewer: ```suggestion
    <td>Server encountered an error while the tenant life cycle status was activated.</td>
```

  </tr>
  <tr>
    <td>TM-65005</td>
    <td>500</td>
    <td>Error building page links</td>

>     <td>Error occurred during building page links. %s</td>

Reviewer: ```suggestion
    <td>Error occurred during building page links.</td>
```

  </tr>
  <tr>
    <td>TM-65006</td>
    <td>501</td>
    <td>Filtering not supported.</td>
    <td>Filtering capability is not supported in this version of the API.</td>
  </tr>
  <tr>
    <td>TM-65007</td>
    <td>500</td>
    <td>Unable to add tenant.</td>
    <td>Error occurred in validating the code.</td>
  </tr>
  <tr>
    <td>TM-65008</td>
    <td>500</td>
    <td>Unable to check availability of domain.</td>
    <td>Server encountered an error while checking for tenant domain</td>
  </tr>
  <tr>
    <td>TM-65009</td>
    <td>500</td>
    <td>Error while deleting the tenant metadata.</td>

>     <td>Server encountered an error while deleting the tenant metadata identified by %s .</td>

Reviewer: identified by ...?
